core/src/main/java/org/apache/accumulo/core/metadata/schema/Ample.java

```diff
@@ -95,6 +95,16 @@ public TableId tableId() {
         throw new UnsupportedOperationException();
       return id;
     }
+
+    public static DataLevel of(TableId tableId) {
+      if (tableId.equals(RootTable.ID)) {
+        return DataLevel.ROOT;
+      } else if (tableId.equals(MetadataTable.ID)) {
+        return DataLevel.METADATA;
+      } else {
+        return DataLevel.USER;
+      }
+    }
```

Comment:
This change makes `Ample` look even more like it's precurser, `MetadataServicer`. `Ample.of(...)` is precisely analogous to `MetadataServicer.forTableId(...)`. :smiley_cat:

One item on my personal wish list (if I ever get time) is to combine these, taking the best of each, and putting them into one.